src/assets/css/styles.css

```diff
@@ -1,26 +1,46 @@
-main {
-    margin-top: 100px;
+html, body {
+    height: 100%;
```

Comment:
Not sure if this line does anything

Reply:
Checked, it has no impact on the page, therefore I have removed it.